### PR TITLE
fix: reducer clusterInfo should not be used

### PR DESCRIPTION
**Pull request by Raubzeug**
Reducer clusterInfo is redundant and should be removed.

### CODE REVIEW

File: src/containers/Header/Header.tsx [hunk 1]
```diff
@@ -33,7 +33,7 @@ function Header() {
         useSelector((state: any) => state);
 
     const clusterName: string = useSelector(
-        (state: any) => state.cluster.data?.Name || state.clusterInfo.title,
+        (state: any) => state.cluster.data?.Name || state.clusterInfo?.title,
```
Comment (ariser) [inline src/containers/Header/Header.tsx:36]: do we really want to keep the usage of a reducer that isn't in the code base anymore?

**Review by ariser — CHANGES_REQUESTED** [review]

**Comment by Raubzeug** [comment]
Yes, now it is needed in internal version.

File: src/containers/Header/Header.tsx (hunk 1 above)
Comment (Raubzeug) [reply src/containers/Header/Header.tsx:36]: Yes, now it is needed in internal version.

Comment (ariser) [reply src/containers/Header/Header.tsx:36]: Keeping in the open source version something that is only required in the internal version doesn't seem right to me. This selector should accept some parameter, and internal version should use it to fallback to `clusterInfo.title`. And in open source we should only keep `state.cluster`.

I can approve this PR if we agree to change this selector in the future

Comment (Raubzeug) [reply src/containers/Header/Header.tsx]: I totally agree with you. It seems not very difficult to achieve this, so I fixed the issue.

**Review by ariser — APPROVED** [review]

**Pull request merged**